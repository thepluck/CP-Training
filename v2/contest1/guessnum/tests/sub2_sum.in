20
79
6
32
35
10
3
23
11
23
5
20
36
42
74
14
19
9
7
41
11
