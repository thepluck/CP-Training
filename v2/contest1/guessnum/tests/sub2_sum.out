1262
10
208
246
23
4
105
25
105
6
83
269
364
1110
42
73
17
11
335
25
